go.mod

```diff
@@ -12,4 +12,5 @@ require (
 	golang.org/x/crypto v0.0.0-20181015023909-0c41d7ab0a0e // indirect
 	golang.org/x/net v0.0.0-20181023162649-9b4f9f5ad519 // indirect
 	golang.org/x/sys v0.0.0-20181023152157-44b849a8bc13 // indirect
+	google.golang.org/appengine v1.2.0 // indirect
```

Comment:
Why did we wind up with an appengine dependency?

Reply:
`go mod vendor` is pulling it in as an indirect dependency from `github.com/go-sql-driver/mysql`

Reply:
I think `go mod vendor` might not be smart enough to know about this build tag?

https://github.com/go-sql-driver/mysql/blob/749ddf1598b47e3cd909414bda735fe790ef3d30/appengine.go#L9

Reply:
Looks like this issue: https://github.com/golang/go/issues/26968 - the exclude tags arg for `go mod tidy` mentioned in the last comment & linked issue isn't available for 1.11.1.